Gems/ROS2/Code/Source/Camera/PostProcessing/ROS2ImageEncodingConvertComponent.h

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#pragma once
+
+#include <AzCore/Component/Component.h>
+#include <AzCore/RTTI/TypeInfoSimple.h>
+#include <ROS2/Camera/CameraPostProcessingRequestBus.h>
+
+namespace ROS2
+{
+    enum class ImageEncoding : AZ::u8
+    {
+        rgba8,
+        rgb8,
+        mono8,
+        mono16,
```

Comment:
Enums in o3de should be [UpperCamelCase](https://github.com/o3de/sig-core/blob/main/governance/Coding-Standards-and-Style-Guide.md#enums).